instance/management/commands/instance_redeploy.py

```diff
@@ -225,6 +225,8 @@ def _pending_instances(self):
                 self.success_tag,
                 self.failure_tag,
             ]
+        ).order_by(
```

Comment:
This very likely doesn't need to have separated lines. The others were separated to allow the whole chain to separate with adding `\`, but if the ending query is so small now that it can probably go on one line.

Reply:
And thanks for the fix :)

Reply:
Oh, @lgp171188 is going to fix this in https://github.com/open-craft/opencraft/pull/368/files.

Reply:
@UmanShahzad I've seem that too. I'll remove these modifications when the PR is approved so I rebase the repo only once.